lib/tasks/teem_tasks.rake

```diff
@@ -0,0 +1,45 @@
+require 'rake'
+
+namespace :teem do
+  desc 'Sync RM users to Teem'
+  task sync_users: :environment do
+    require 'teem.rb'
+
+    token = Teem.teem_authorize()
+    teem_groups = Teem.teem_get_groups(token)
+
+    count = 1
+
+    # Need to add members: Looping through rm people and comparing with all in Teem.
+    Application.find_by(name: "Teem Calendars").roles.each do |role|
```

Comment:
If 'Teem Calendars' does not exist, we'll have a null pointer exception. Best break this up into two lines:

`teem_application = Application.find_by(name: "Teem Calendars")
if teem_application
   teem_application.roles.each ...
else
   STDERR.puts "Cannot proceed, required application Teem Calendars not found"
   exit(-1)
end`